common/src/Model/GameConfig.cpp

```diff
@@ -165,11 +166,54 @@ namespace TrenchBroom {
             return flags == other.flags;
         }
 
+        FaceAttribsDefaultsConfig::FaceAttribsDefaultsConfig() :
+        textureName(BrushFace::NoTextureName),
+        offset(vm::vec2f::zero()),
+        scale(vm::vec2f(1.0f, 1.0f)),
+        rotation(0.0f),
+        surfaceContents(0),
+        surfaceFlags(0),
+        surfaceValue(0.0f) { }
+
+        FaceAttribsDefaultsConfig::FaceAttribsDefaultsConfig(const std::string& i_textureName,
+                                                             const vm::vec2f& i_offset,
+                                                             const vm::vec2f& i_scale,
+                                                             const float& i_rotation,
+                                                             const int& i_surfaceContents,
```

Comment:
minor nitpick, these should be just `float` instead of `const float&` and `int` instead of `const int&` (other types should stay as const reference)